vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/ApiRegisterUser.java

```diff
@@ -0,0 +1,43 @@
+package fr.insalyon.creatis.vip.api.business;
+
+import fr.insalyon.creatis.vip.api.exception.ApiException;
+import fr.insalyon.creatis.vip.core.client.bean.User;
+import fr.insalyon.creatis.vip.core.server.business.BusinessException;
+import fr.insalyon.creatis.vip.core.server.business.ConfigurationBusiness;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author khalilkes service to signup a user in VIP
+ */
+@Service
+public class ApiRegisterUser {
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private final Environment env;
+    private final ConfigurationBusiness configurationBusiness;
+
+    public ApiRegisterUser(Environment env, ConfigurationBusiness configurationBusiness) {
+        this.env = env;
+        this.configurationBusiness = configurationBusiness;
+    }
+
+    /**
+     *
+     * @param user
+     * @param comments
+     * @param accountTypes
+     * @throws ApiException
+     */
+    public void signup(User user, String comments, String[] accountTypes) throws ApiException {
+        try {
+            configurationBusiness
+                    .signup(user, comments, accountTypes);
```

Comment:
Why not use the `signup` method with the `automaticCreation` option ? This way it will not send an email to the user.